cmd/antrea-controller/controller.go

```diff
@@ -53,6 +53,9 @@ import (
 // https://github.com/kubernetes/kubernetes/blob/release-1.17/pkg/controller/apis/config/v1alpha1/defaults.go#L120
 const informerDefaultResync = 12 * time.Hour
 
+// Certificate takes a year to expire. Refresh certificate an hour before that.
+const rotationInterval = 8759 * time.Hour
```

Comment:
maybe we can use `time.Hour * (24 * 365 - 1)` to match https://github.com/kubernetes/client-go/blob/00dbcca6ee44c678754d3f5fda1bd0e704b26fe2/util/cert/cert.go#L96